msteams-platform/toolkit/teams-toolkit-overview-visual-studio.md

```diff
@@ -0,0 +1,109 @@
+
+You can also access Teams Toolkit menu from Solution Explorer.
+
+4. Right-click on your **Project**.
+5. Select **Teams Toolkit** > **Teams Toolkit menu** options.
+
+   :::image type="content" source="../assets/images/teams-toolkit-overview/teams-toolkit-operations-menu1.png" alt-text="Teams toolkit operations from Project":::
+
+   > [!NOTE]
+   > In this scenario the project name is **MyTeamsApp1**.
+
+You can perform the following functions on Teams Toolkit for Visual Studio:
+
+|Function  |Description  |
+|---------|---------|
+|Create Teams Project     |Create Teams project using Teams template in Visual Studio         |
+|Prepare Teams App Dependencies     |Before you do a local debug perform this step, it helps you to set up the local debug dependencies and register Teams app in Teams platform. You need a Microsoft 365 account. For more information, see [Debug your Teams app locally using Visual Studio](debug-teams-app-visual-studio.md)         |
+|Open Manifest File     |To open Teams manifest file, you can hover over the parameters to preview the values. For more information, see [Edit Teams app manifest using Visual Studio](VS-TeamsFx-preview-and-customize-app-manifest.md)         |
+|Update Manifest in Teams Developer Portal     |When you update the manifest file, only then you can redeploy the manifest file to Azure without deploying the whole project again. Use this command to update your changes to remote. For more information, see [Edit Teams app manifest using Visual Studio](VS-TeamsFx-preview-and-customize-app-manifest.md)       |
+|Provision to the Cloud     |This option helps you to create Azure resources that host your Teams app. For more information, see [Provision cloud resources using Visual Studio](provision VS.md)         |
+|Deploy to the Cloud     |This option helps you to copy your code to the Azure resources created when you did “Provision to the Cloud”. For more information, see [Deploy Teams app to the cloud using Visual Studio](deploy to the cloud for VS.md)         |
+|Preview in Teams     |This option launches the Teams web client and lets you preview Teams app in their browser.         |
+|Zip App Package     |This option generates a Teams app package in the `Build` folder under the project. You can upload the package to Teams client and run the Teams app.         |
+
+The following operations aren't supported in Teams Toolkit for Visual Studio yet compared to Teams Toolkit for Visual Studio Code, however they're planned in the future product road map.
+
+* Add another Teams capabilities to your Teams app.
+* Add more Azure resources to your Teams app
+* Add single sign-on to your Teams app.
+* Add API connection to your Teams app.
+* Customize Azure AD manifest.
+* Add CI/CD pipelines.
+* Manage multiple cloud environments.
+* Collaborate on Teams projects.
+* Publish Teams app.
+
```

Comment:
Add a new section here -
TeamsFx .NET SDK Reference docs:
https://docs.microsoft.com/dotnet/api/Microsoft.Extensions.DependencyInjection
https://docs.microsoft.com/dotnet/api/Microsoft.TeamsFx
https://docs.microsoft.com/dotnet/api/Microsoft.TeamsFx.Configuration
https://docs.microsoft.com/dotnet/api/Microsoft.TeamsFx.Conversation
https://docs.microsoft.com/dotnet/api/Microsoft.TeamsFx.Helper

Reply:
The section has been added along with the links.